layouts/shortcodes/events/sponsors.html

```diff
@@ -0,0 +1,79 @@
+<!-- 
```

Comment:
This is for https://github.com/EclipseFdn/ecdtools.eclipse.org/issues/96#issuecomment-797022680 "4: Under the 'Organized By' section, please add the following phrase above the ECD Tools logo:" 

I temporarily choose to use override, while I am not sure if I need to add this feature into the base theme. 🤔 

What do you think?

Reply:
I only added a 

```
  <div class="eclipsefdn-registration-body">
    {{ .Inner | markdownify }}
  </div>
```

Reply:
We can add it to the base theme but please make sure to only include the div if .Inner is not empty